doc/site/zh-cn/guides/thread-primer/node-roles-and-types.md

```diff
@@ -0,0 +1,125 @@
+# 节点角色和类型
+
+## 转发角色
+
+<figure class="attempt-right">
+<a href="../images/ot-primer-roles_2x.png"><img src="../images/ot-primer-roles.png" srcset="../images/ot-primer-roles.png 1x, ../images/ot-primer-roles_2x.png 2x" border="0" alt="OT Node Roles" /></a>
+</figure>
+
+在 Thread 网络中，节点分成两种转发角色：Router 和 End Device。
+
+### Router
+
+Router 节点的行为如下：
+
+* 为网络设备转发数据包
+* 为尝试加入网络的设备提供安全的 commissioning 服务
+* 始终打开它的收发器
+
+### End Device
+
+End Device 节点的行为如下：
+
+* 主要与单个 Router 进行通信
+* 不会为其他网络设备转发数据包
+* 可以关闭它的收发器来降低功耗
+
+Key Point: Router 和 End Device 之间的关系称为父子关系。End Device 正确地依附到一个 Router 上，其中 Router 始终作为父节点，End Device 则始终是子节点。
+
+## 设备类型
+
+此外，节点有许多种类型。
+
+<figure class="attempt-right">
+<a href="../images/ot-primer-taxonomy_2x.png"><img src="../images/ot-primer-taxonomy.png" srcset="../images/ot-primer-taxonomy.png 1x, ../images/ot-primer-taxonomy.png 2x" border="0" alt="OT Device Taxonomy" /></a>
+</figure>
+
+### Full Thread Device
+
+一个 FTD（Full Thread Device）总是打开它的无线电收发器，它订阅所有 Router 的多播地址，并维护 IPv6 地址映射。FTD 有三种类型：
+
+* Router
+* REED（Router Eligible End Device）— 可以升级为 Router
+* FED（Full End Device）— 无法升级为 Router
+
+FTD 可以作为 Router（父）或 End Device（子）。
+
+### Minimal Thread Device
+
+MTD（Minimal Thread Device）不会订阅多播流量，并且它会将它的所有消息发送给它的父节点。MTD 有两种类型：
```

Comment:
The original english seems not accurate @jwhui @Vyrastas 
> A Minimal Thread Device does not subscribe to multicast traffic 

e.g. All Thread devices (including MTD) subscribes mesh local prefix based multicast address.

Reply:
I also think there is a conflict here.
`A Minimal Thread Device does not subscribe to multicast traffic and forwards all messages to its Parent.`
`There is an additional unicast prefix-based multicast address used for All Thread Nodes, including SEDs. `

Reply:
Whatever it should be, @dsyx should use.  In tandem with this PR, let's create an Issue with all the updates needed for the English version, and I can handle that in a separate PR.

Reply:
I think the intended text is something like: "A Minimal Thread Device does not subscribe to the all-routers multicast address and forwards all messages to its Parent."

Reply:
Agree to track it separately @Vyrastas

Reply:
#5153